src/pydap/model.py

```diff
@@ -290,13 +290,13 @@ class StructureType(DapType, Mapping):
     def __init__(self, name, attributes=None, **kwargs):
         super(StructureType, self).__init__(name, attributes, **kwargs)
 
-        # emulate a simple ordered dict
-        self._keys = []
+        # allow some keys to be hidden:
+        self._visible_keys = []
```

Comment:
Good point, I'll change it. How do you like the new approach to dealing with keys?